planning/behavior_velocity_intersection_module/src/scene_intersection.cpp

```diff
@@ -934,7 +934,8 @@ bool IntersectionModule::checkCollision(
   using lanelet::utils::getPolygonFromArcLength;
 
   const auto & objects_ptr = planner_data_->predicted_objects;
-
+  const auto lanelets_on_path = planning_utils::getLaneletsOnPath(
+    path, planner_data_->route_handler_->getLaneletMapPtr(), planner_data_->current_odometry->pose);
```

Comment:
Can you move this variable to just before your other changes since it is not used anywhere earlier